compiler/parser/chapel.ypp

```diff
@@ -325,6 +325,7 @@
 %nonassoc TPRAGMA
 
 %type <flag> opt_config
+%type <b> access_control
```

Comment:
Yeah, it seemed like the name would either be too long with opt in front, or imprecise if I shortened it.